src/__stories__/display-media-card-story.tsx

```diff
@@ -76,41 +84,105 @@ export const Default: StoryComponent<DisplayMediaCardArgs> = ({
+            ? {
+                  onClose: closable ? () => {} : undefined,
+                  actions: withTopAction
+                      ? [
+                            {
+                                Icon: IconLightningRegular,
+                                onPress: () => {},
+                                label: 'Lightning',
+                            },
+                        ]
+                      : undefined,
+                  backgroundImage: 'test',
+              }
+            : {
+                  backgroundVideo: 'test',
+                  poster: 'test',
+              };
+
     return (
-        <DisplayMediaCard
-            onClose={closable ? () => {} : undefined}
-            actions={
-                withTopAction
-                    ? [
-                          {
-                              Icon: IconLightningRegular,
-                              onPress: () => {},
-                              label: 'Lightning',
-                          },
-                      ]
-                    : undefined
-            }
-            backgroundImage={BACKGROUND_SRC}
-            icon={icon}
-            headline={headline ? <Tag type={headlineType}>{headline}</Tag> : undefined}
-            pretitle={pretitle}
-            title={title}
-            description={description}
-            button={button}
-            buttonLink={buttonLink}
-            secondaryButton={secondaryButton}
-            dataAttributes={{testid: 'display-media-card'}}
-            aria-label="Display data card label"
-            width={width}
-            aspectRatio={aspectRatio}
-        />
+        <Stack space={32} dataAttributes={{testid: 'display-media-card'}}>
+            <DisplayMediaCard
+                {...backgroundProps}
+                icon={icon}
+                headline={headline ? <Tag type={headlineType}>{headline}</Tag> : undefined}
+                pretitle={pretitle}
+                title={title}
+                description={description}
+                button={button}
+                buttonLink={buttonLink}
+                secondaryButton={secondaryButton}
+                aria-label="Display media card label"
+                width={width}
+                aspectRatio={aspectRatio}
+            />
+
+            <Title1>Wrong source for media</Title1>
+            <DisplayMediaCard
+                {...wrongBackgroundProps}
+                icon={icon}
+                headline={headline ? <Tag type={headlineType}>{headline}</Tag> : undefined}
+                pretitle={pretitle}
+                title={title}
+                description={description}
+                button={button}
+                buttonLink={buttonLink}
+                secondaryButton={secondaryButton}
+                aria-label="Display media card fallback label"
+                width={width}
+                aspectRatio={aspectRatio}
+            />
+
+            <Title1>Wrong source for media with inverse</Title1>
```

Comment:
can you include a backgroundBrand in this case?
![image](https://user-images.githubusercontent.com/6722153/236147853-30a79b4d-5b2f-4c16-a873-2f2cb596b7ee.png)